appinventor/blocklyeditor/src/drawer.js

```diff
@@ -81,6 +87,209 @@ Blockly.Drawer.buildTree_ = function() {
   return tree;
 };
 
+Blockly.Drawer.createSubsetBlockInfoArray_ = function() {
+  console.log("HELLO creating subset block info array here");
+  try {
+    console.log("trying to set subset string");
+    var subsetJsonString;
+    var formName = Blockly.mainWorkspace.formName;
+    var screenName = formName.split("_")[1];
+    if (window.parent.BlocklyPanel_getComponentInstancePropertyValue) {
+      subsetJsonString = window.parent.BlocklyPanel_getComponentInstancePropertyValue(formName, screenName, "BlocksToolkit");
+    } else {
+      subsetJsonString = JSON.stringify(window.reactGetSubsetString());
```

Comment:
I don't see where window.reactGetSubsetString is defined?

Reply:
Huh.  I didn't immediately find it either. I'll poke around at it. This is Lynda's code, but I never had a problem with it.

Console logs should probably be cleaned up a bit too.